modules/ingestor/tests/performance/csaf.rs

```diff
@@ -0,0 +1,56 @@
+#![allow(clippy::expect_used)]
```

Comment:
We can remove this line

![2024-08-27_06-12](https://github.com/user-attachments/assets/82716283-3404-43b4-b1c1-04ce4d5373f7)

Reply:
Change pushed.